updating/migrating-to-multi-payload.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="migrating-clusters-to-multi-payload"]
+= Migrating to a cluster with multi-architecture compute machines
+include::_attributes/common-attributes.adoc[]
+:context: updating-clusters-overview
+
+toc::[]
+
+You can migrate your current cluster with single-architecture compute machines to a cluster with multi-architecture compute machines by updating to a multi-arch manifest-listed payload. This allows you to add mixed architecture compute nodes to your cluster. 
+
+For information about configuring your multi-architecture compute machines, see _Configuring multi-architecture compute machines on an {product-title} cluster_.
+
+[IMPORTANT]
+====
+Once a cluster has transitioned to using a multi-architecture payload, it can no longer return back to cluster with single-architecture compute machines or accept a single-architecture upgrade payload. 
```

Comment:
> it can no longer return back to a cluster with single-architecture compute machines

Hmm, seems like it can no longer return back to a single-architecture upgrade payload, but it can be a cluster with single-architecture compute machines. 

@aleskandro wdyt?

Reply:
i think it is enough to say something like:
"migration from a multi-arch payload to a single arch payload is not supported, although it can be done through an explicit upgrade by specifying the image pullspec"

Reply:
@Prashanth684 Where can a user learn more about scaling down machinesets? And how would they verify which architectures match the architecture of their control plane?

Reply:
The command below can be used to find the architecture of the master nodes:

`oc get nodes -l node-role.kubernetes.io/master -o 'jsonpath={.items[0].status.nodeInfo.architecture}'` 

This is the doc for manually scaling up/down machinesets:

https://docs.openshift.com/container-platform/4.12/machine_management/manually-scaling-machineset.html

Reply:
> migration from a multi-arch payload to a single arch payload is not supported, although it can be done...

Many things that are not supported *can* be done.  What do we gain by documenting them?   It seems like distracting noise for folks sticking to supported paths.  And do we want folks following official docs while blazing unsupported and untested trails?  I'd rather say "...  is not supported.", and leave it at that.

Reply:
thank you :)